### PR TITLE
added selection sort using js

### PR DESCRIPTION
**Pull request by Sloth-Panda**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES
(Write your answer here.)

### Description

Added description of selection sort using js with describing what selection sort is.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
corresponds to issue -> #2075 
(Write your answer here.)

### CODE REVIEW

**Comment by Sloth-Panda** [comment]
@atarax665 @HarshCasper kindly add DWOC and level tags to this pr.

**Comment by Sloth-Panda** [comment]
@shraddhavp can you please review the pr and kindly add dwoc and level tags

File: JavaScript/sort/selectionSort.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
```
Comment (atarax665) [inline JavaScript/sort/selectionSort.js:1]: We already have PR under review for selection sort in Js. #2091  Please wait for us to close or merge that PR.

Comment (Sloth-Panda) [reply JavaScript/sort/selectionSort.js:1]: But I created the issue and someone else just came and made a Pr , should it be like that? Is there any meaning in assigning anyone any issue then ? @atarax665

**Pull request closed without merging**